docs/components/dropdown.md

```diff
@@ -0,0 +1,568 @@
+  :items="items"
+/>
+```
+
+### showCaret
+
+Use this prop if you would like the trigger button to display the caret.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :items="deepClone(defaultItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  show-caret
+  :items="items"
+/>
+```
+
+### icon
+
+:::warning NOTE
+This prop will be removed in the `9.0.0-beta.0` release.
+:::
+
+A string for the `KIcon` to be displayed on the dropdown button with or in place of the button label.
+
+<ClientOnly>
+  <KDropdown icon="cogwheel" :items="deepClone(defaultItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown icon="cogwheel" :items="items" show-caret />
+```
+
+### width
+
+The width of the dropdown body (defaults to `auto`). Currently we support numbers (will be converted to `px`), `auto`, and percentages for width.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :items="deepClone(defaultItems)" width="700" />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  width="700"
+  :items="items"
+/>
+```
+
+### kpopAttributes
+
+Use the `kpopAttributes` prop to configure the KPop [props](/components/popover) dropdown.
+
+<ClientOnly>
+  <KDropdown trigger-text="Documentation" :kpop-attributes="{ placement: 'leftStart' }" :items="deepClone(defaultItems)" />
+</ClientOnly>
+
+```html
+<KDropdown 
+  trigger-text="Documentation"
+  :kpop-attributes="{ placement: 'leftStart' }"
+  :items="items"
+/>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with [`disabledTooltip` prop](#disabledtooltip).
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<ClientOnly>
+  <KDropdown
+    trigger-text="Documentation"
+    disabled
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItems)"
+  />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Documentation"
+  disabled
+  disabled-tooltip="You can't click me"
+  :items="items"
+/>
+```
+
+### isSelectionMenu
+
+Use this prop when a visual indication of the currently selected menu item is needed. 
+
+By default the dropdown has no notion of "selection":
+
+<ClientOnly>
+  <KDropdown trigger-text="Select region" :items="deepClone(selectionMenuItems)" show-caret />
+</ClientOnly>
+
+```html
+<KDropdown
+  trigger-text="Select region"
+  :items="[{ label: 'US (United States)', value: 'us' }, { label: 'FR (France)', value: 'fr' }]"
+  show-caret
+/>
+```
+
+When `isSelectionMenu` is `true`, `selected` state is handled automatically when clicking a KDropdownItem if used in conjunction with the `items` prop. Each item should have a `label` and a `value`. To keep track of changes in your host app you can utilize [`@change` event](#events).
+
+<ClientOnly>
+  <KDropdown
+    is-selection-menu
+    :items="selectionMenuItems"
+    trigger-text="Select region"
+    @change="handleSelectionMenuUpdate"
+    show-caret
+  />
+</ClientOnly>
```

Comment:
so you're saying by default it will have the same (primary) appearance as without the prop? 

Yes this will **definitely** need to be documented; I know it will break the App Shell 😬

Reply:
Will add it in adoption tickets description.

Reply:
It will need to be in the migration document as well here since it's technically a breaking change

Reply:
Added a sentence about this to migration doc